apps/explorer/src/components/activity/Activity.tsx

```diff
@@ -140,10 +139,9 @@ export function Activity({ initialLimit, disablePagination }: ActivityProps): JS
                         </>
                     ) : null*/}
                     {/* todo: re-enable this when rpc is stable */}
-                    {pollingTxnTableEnabled &&
-                        selectedCategory === ActivityCategory.Transactions && (
-                            <PlayPause paused={paused} onChange={handlePauseChange} />
-                        )}
+                    {selectedCategory === ActivityCategory.Transactions && (
```

Comment:
I dont think we should remove `pollingTxnTableEnabled` from there as it determines the feature being present or not